site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +244,547 @@ They can do the same for Chrome on desktop by going to
    width="800", height="678", class="screenshot"
 %}
 
-## How can IdPs support FedCM? {: #support-fedcm}
+## Support and compatibility {: #compatibility }
+  
+FedCM is supported in Chrome on all platforms.
+
+FedCM isn't supported by other browsers yet, but Mozilla is [implementing a
+prototype](https://bugzilla.mozilla.org/show_bug.cgi?id=1782066) to Firefox and
+[WebKit team has expressed general support and interest in working together on
+the FedCM
+proposal](https://lists.webkit.org/pipermail/webkit-dev/2022-March/032162.html).
+
+## How can identity providers develop FedCM?
+
+You need a secure context (HTTPS or localhost) in Chrome to use the Federated
+Credential Management API.
+
+### Debug code on Chrome on Android {: #remote-debug-android}
+
+Set up and run a server locally to debug your FedCM code. You can [access this
+server in Chrome on an Android device connected using a USB cable with port
+forwarding](/docs/devtools/remote-debugging/local-server/).
+
+You can use DevTools on desktop to debug Chrome on Android by following the
+instructions at [Remote debug Android
+devices](/docs/devtools/remote-debugging/).
+
+## Use the FedCM API {: #use-api }
+
+You integrate with FedCM by creating [a well-known file, config file and
```

Comment:
you mentioned well-known file first here, perhaps follow the same order below?

Reply:
Swapped the order.